core/trino-spi/src/main/java/io/trino/spi/security/ViewExpression.java

```diff
@@ -36,21 +38,25 @@ public ViewExpression(String identity, Optional<String> catalog, Optional<String
         }
     }
 
+    @Internal
     public String getIdentity()
     {
         return identity;
     }
 
+    @Internal
```

Comment:
I generally like the idea of being able to say which "direction" does given class work.

How would we enforce our connectors do not call internal APIs?
How should external connectors' maintainers to that?

In this particular case, what if one connector needs, for some reason, to rewrap a ViewExpression returned by some other connector. Can it do that? Is it illegal?